tests/unit/einstein/test_ceph.py

```diff
@@ -0,0 +1,170 @@
+import time
+import unittest
+import subprocess
+from ims.common import config
+
+config.load()
+
+import rados
+import rbd
+from ims.common import constants
+from ims.einstein import ceph
+from ims.einstein.operations import BMI
+from ims.exception import file_system_exceptions
+from ims.common.log import trace
+import logging
+
+_cfg = config.get()
+
+POOL = _cfg.fs.pool
+# test image
+CEPH_IMG = "BMI_TEST_CEPH"
+# test image size
+CEPH_IMG_SIZE = 1024
+# test snap_image
+CEPH_EXISTING_IMG = "bmi_ci.img"
+CEPH_SNAP_IMG = "BMI_TEST_CEPH_SNAP"
+# test write
+TEST_DATA = 'bmi' * 10
+OFFSET = 0
+# test clone
+CEPH_CHILD_IMG = "BMI_TEST_CEPH_CHILD_IMAGE"
+
+
+class TestImage(unittest.TestCase):
+    @trace
+    def setUp(self):
+        self.fs = ceph.RBD(_cfg.fs, _cfg.iscsi.password)
+        self.fs.create_image(CEPH_IMG, CEPH_IMG_SIZE)
+        self.TEST_NOT_EXIST_IMG_NAME = "BMI_TEST_IMG_NOT_FOUND"
+
+    def TestGetImage(self):
```

Comment:
Function names should be in lowercase like test_get_image. Change all the function names